public/index.html

```diff
@@ -0,0 +1,370 @@
+          </figure>
+        </section>
+        <section class="strengths">
+          <h2>DEVELOPMENT</h2>
+          <figure>
+            <span class="icon-gear"></span>
+            <figcaption>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</figcaption>
+          </figure>
+        </section>
+      </section>
+      <section class="price-tables">
+        <h3>PRICING TABLES</h3>
+        <table>
+          <caption>BASIC</caption>
+          <tr><th scope="col" class="price"><strong>$5.99</strong> / MONTH</th></tr>
+          <tr><td>3 Projects</td></tr>
+          <tr><td>2 Users</td></tr>
+          <tr><td>1 Gb user</td></tr>
+          <tr><td>1 Calendar</td></tr>
+          <tr><td class="signup"><button>SIGN UP</button></td></tr>
+        </table>
+        <table>
+          <caption>STANDARD</caption>
+          <tr><th scope="col" class="price"><strong>$10.99</strong> / MONTH</th></tr>
+          <tr><td>7 Projects</td></tr>
+          <tr><td>5 Users</td></tr>
+          <tr><td>10 Gb user</td></tr>
+          <tr><td>5 Calendar</td></tr>
+          <tr><td class="signup"><button>SIGN UP</button></td></tr>
+        </table>
+        <table>
+          <caption>PROFESSIONAL</caption>
+          <tr><th scope="col" class="price"><strong>$25.99</strong> / MONTH</th></tr>
+          <tr><td>15 Projects</td></tr>
+          <tr><td>50 Users</td></tr>
+          <tr><td>100 Gb user</td></tr>
+          <tr><td>25 Calendar</td></tr>
+          <tr><td class="signup"><button>SIGN UP</button></td></tr>
+        </table>
+        <table>
+          <caption>PREMIUM</caption>
+          <tr><th scope="col" class="price"><strong>$49.99</strong> / MONTH</th></tr>
+          <tr><td>UNLIMITED Projects</td></tr>
+          <tr><td>UNLIMITED Users</td></tr>
+          <tr><td>1000 Gb user</td></tr>
+          <tr><td>UNLIMITED Calendar</td></tr>
+          <tr><td class="signup"><button>SIGN UP</button></td></tr>
+        </table>
+      </section>      
+    </article>
+    <span class="orange-notch"></span>
+    <!-- portfolio  -->
+    <article class="portfolio clearfix">
+      <h1>PORTFOLIO</h1>
+      <img src="images/portfolio.jpg" alt="Iphone held in Times Square">
+      <nav class="nav secondary-nav clearfix">
+        <ul class="flex-secnav">
+          <li><a class="flex-secnav-item-1" href="#">All works</a></li>
+          <li><a class="flex-secnav-item-2" href="#">Webdesign</a></li>
+          <li><a class="flex-secnav-item-5" href="#">Brand &amp; Logotypes</a></li>
+          <li><a class="flex-secnav-item-3" href="#">UI design</a></li>
+          <li><a class="flex-secnav-item-4" href="#">Icons design</a></li>
+        </ul>
+      </nav>
+      <section class="gallery-flex clearfix">
```

Comment:
I don't think a `section` is appropriate here, since there's only one in this `article` (it could be argued that multiple thematic chunks in an article would warrant `section`s, but here there only one thematic section) Again, if you need an element for styling purposes, use a `div`.